data/aerial-photography/index.html

```diff
@@ -13,12 +13,13 @@
 title: Aerial Photography Products for Utah
 categories: []
 ---
-<p>All aerial photography downloads available from AGRC, with the exception of the Google imagery, have a UTM NAD83 zone 12 north meters projection.</p>
-<p>Common problems and issues people encouter with the imagery products are <a href="{{ '/data/aerial-photography/imagery-problems/' | prepend: site.baseurl }}">compiled into a document</a> with descriptions and workarounds. <strong>If you are having issues, please look for your issue there first</strong>. {% include contact.html subject=page.title contact=site.data.contacts.imagery %}</p>
+<p>All aerial photography downloads available from AGRC, with the exception of Google Imagery, have a UTM NAD83 zone 12 north meters projection.</p>
```

Comment:
Could we add some punctuation to "UTM NAD83 zone 12 north meters projection"? And should we spell out those acronyms? As worded, I don't know what this phrase means. 

Also, can we have more of a transition between these first two paragraphs? Right now, they seem a bit disjointed. I'm not sure how the two ideas are related.

Reply:
`UTM NAD83 zone 12 north meters projection` is a well known item for our user base. It is expanded to Universal Transverse Mercator North American Datum 1983 Zone 12 North Meters Projection. It's a state standard and should not need punctuation or expansion in my opinion.